core/domain/skill_services_test.py

```diff
@@ -515,8 +515,8 @@ def test_get_skill_from_model_with_invalid_misconceptions_schema_version(
             rubrics=[],
             next_misconception_id=0,
             misconceptions_schema_version=0,
-            rubric_schema_version=1,
-            skill_contents_schema_version=1,
+            rubric_schema_version=3,
```

Comment:
Why is this 3 (from 1)?

Reply:
This actually doesn't matter, this will work even if the version was 2.